pkgs/by-name/qq/qqsp/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  adwaita-qt,
+  libadwaita,
+  adw-gtk3,
+  gst_all_1,
+  gnome-themes-extra,
+  librsvg,
+  fetchgit,
+  qt5,
+}:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
Related to this, is stdenv.mkDerivation correct for QT apps? It seems to work, but I've seen various references to "qt5.callPackage"